aws/iam/datadog/main.tf

```diff
@@ -0,0 +1,74 @@
+resource "aws_iam_policy" "mod" {
+  name        = "${var.datadog_policy_name}"
+  description = "${var.datadog_policy_name}"
+  path        = "/"
+
+  policy = <<JSON
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "autoscaling:Describe*",
+        "cloudtrail:DescribeTrails",
+        "cloudtrail:GetTrailStatus",
+        "cloudwatch:Describe*",
+        "cloudwatch:Get*",
+        "cloudwatch:List*",
+        "ec2:Describe*",
+        "ec2:Get*",
+        "ecs:Describe*",
+        "ecs:List*",
+        "elasticache:Describe*",
+        "elasticache:List*",
+        "elasticloadbalancing:Describe*",
+        "elasticmapreduce:List*",
+        "iam:Get*",
+        "iam:List*",
+        "kinesis:Get*",
+        "kinesis:List*",
+        "kinesis:Describe*",
+        "logs:Get*",
+        "logs:Describe*",
+        "logs:TestMetricFilter",
+        "rds:Describe*",
+        "rds:List*",
+        "route53:List*",
+        "s3:GetBucketTagging",
+        "ses:Get*",
+        "ses:List*",
+        "sns:List*",
+        "sns:Publish",
+        "sqs:GetQueueAttributes",
+        "sqs:ListQueues",
+        "sqs:ReceiveMessage"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    }
+  ]
+}
+JSON
+}
```

Comment:
We have a story in the backlog to use the actual AWS managed ReadOnly policy where appropriate. I think we should use something like I did in https://github.com/counterfind/terraform/blob/master/iam_datadog.tf#L29

Reply:
I don't like that because it isn't the actual policy suggested by DataDog. It gives a lot more rights then are necessary. https://docs.datadoghq.com/integrations/amazon_web_services/#installation

Reply:
ok I am a bit worried that we wont be notified when we need to expand the access for this role, but I agree that you are correct.